src/main/kotlin/com/hibob/KotlinBasics/extentions.kt

```diff
@@ -0,0 +1,22 @@
+package com.hibob.KotlinBasics
+//Q1
+fun List<Int>.sum():Int{
+    var result = 0
+    for (i in this){
+        result+=i
+    }
+    return result
+}
+
+fun main(args:Array<String>){
```

Comment:
you can delete it in the pr it for your checks only

Reply:
Got it!
Thanks for the review Lior